.pipelines/scripts/run-e2e.sh

```diff
@@ -72,16 +72,16 @@ fi
 if [[ -z "${CLUSTER_CONFIG_PATH:-}" ]]; then
   CLUSTER_CONFIG_PATH="${REPO_ROOT}/.pipelines/templates/basic-lb.json"
   if [[ "${CLUSTER_TYPE:-}" == "autoscaling" ]]; then
-    CLUSTER_CONFIG_PATH="${REPO_ROOT}/.pipelines/templates/autoscaling.json"
+    CLUSTER_CONFIG_PATH="${REPO_ROOT}/kubetest2-aks/cluster-templates/autoscaling.json"
```

Comment:
If you want to use your cluster_config, please add your new json file into the repo and set `CLUSTER_CONFIG_PATH`.
In order to build image/release properly, we need to run upstream AKS pipeline with all branches but only master branch has `kubetest2-aks/` path.

Reply:
I will revert the current change, and continue to follow your advice to use CLUSTER_CONFIG_PATH for the pipeline..